apps/designer/tsconfig.json

```diff
@@ -2,9 +2,20 @@
   "extends": "../../node_modules/@webstudio-is/tsconfig/remix.json",
   "include": ["remix.env.d.ts", "**/*.ts", "**/*.tsx"],
   "compilerOptions": {
+    "composite": true,
     "paths": {
       "~/*": ["app/*"]
     },
-    "baseUrl": "."
-  }
+    "baseUrl": ".",
+    "rootDir": "./app"
+  },
+  "references": [
```

Comment:
would have to add every new package though, can't it use some asterix packages/* or something?

Reply:
ye, this is the problematic part - and why I've initially wanted to use a different strategy, that different strategy would require some additional scripts and stuff and would take me more time to prepare. I've figured out that it's better to roll with this strategy, for now, to get improvements ASAP than to wait on that other strategy.

We can't use the asterisk here because the listed packages allow TS to build the dependency graph (it's somewhat weird but it can't figure this out on its own using just `package.json`) and the dep graph it needed here to "build" those packages in the correct order.

We could scriptify regenerating those references in a precommit hook or something. That would mitigate the maintainability concern.

Reply:
we should def. have a script and we should check if someone else has solved that before, because it sounds like there must be something, maybe even specifically for turborepo?